src/components/auth/SocialLogin.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
```

Comment:
Since you are using a functional component, there is no need to call Component here, you can just import it this way `import React from 'react';`